editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid

```diff
@@ -32,42 +32,131 @@ The ''action-createtiddler'' widget is invisible.
 <<.from-version "5.2.0">> The content of the <<.wid "action-createtiddler">> widget is executed after the new tiddler has been created. The title of the newly created tiddler is stored in the variable <<.var "createTiddler-title">>.
 
 |!Variables |!Description |
-|`createTiddler-title` |The tittle of the tiddler that has been created. See [[ActionCreateTiddlerWidget Example 5]] |
+|`createTiddler-title` |The tittle of the tiddler that has been created. See the 5th example |
```

Comment:
Why replace the link with the clumsy textual reference?

Reply:
Usually action-widget doc tiddlers contain their examples in the doc tiddler. ... That's why I did remove them for consistency reasons.

Reply:
Also the numbering does not work well, if the order should be changed. If we do have 1 tiddler per example they should have better names -- other than 1, 2, 3